### ISSUE TITLE
relation "app_client" does not exist

### ISSUE THREAD

**Issue by wiel** [bernardopires/django-tenant-schemas#236]
Traceback (most recent call last):
      File "<console>", line 1, in <module>
      File "c:\Python27\lib\site-packages\tenant_schemas\models.py", line 57, in save
        self.delete(force_drop=True)
      File "c:\Python27\lib\site-packages\tenant_schemas\models.py", line 74, in delete
        super(TenantMixin, self).delete(*args, **kwargs)
      File "c:\Python27\lib\site-packages\django\db\models\base.py", line 695, in delete
        collector.delete()
      File "c:\Python27\lib\site-packages\django\db\models\deletion.py", line 277, in delete
        query.delete_batch(pk_list, self.using)
      File "c:\Python27\lib\site-packages\django\db\models\sql\subqueries.py", line 49, in delete_batch
        self.do_query(self.get_meta().db_table, where, using=using)
      File "c:\Python27\lib\site-packages\django\db\models\sql\subqueries.py", line 34, in do_query
        self.get_compiler(using).execute_sql(None)
      File "c:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 786, in execute_sql
        cursor.execute(sql, params)
      File "c:\Python27\lib\site-packages\django\db\backends\util.py", line 53, in execute
        return self.cursor.execute(sql, params)
      File "c:\Python27\lib\site-packages\django\db\utils.py", line 99, in __exit__
        six.reraise(dj_exc_type, dj_exc_value, traceback)
      File "c:\Python27\lib\site-packages\django\db\backends\util.py", line 53, in execute
        return self.cursor.execute(sql, params)
    ProgrammingError: relation "custom_client" does not exist
    LINE 1: DELETE FROM "custom_client" WHERE "id" IN (2)

I can't seem to resolve this error even after trying the solutions in #226

**Comment by m4l** [comment]
This can be hard to trace, for me it was a signal handler (post_syncdb, post_save, etc). Code that's triggered 'in between' is possibly changing the schema of the current connection (away from public). If you have any such signal handlers, disable and try running it again.

**Comment by wiel** [comment]
I was debugging with Pdb a while ago and saw that the problem was on my side. south threw an AttributeError exception containing the text 'migrations'. I googled around and now Im guessing that one of my apps does not have a migration folder. after i disabled south I was able to create the non public tenant. i'll try to resolve the migration folder issue on my own at a later date.

the exception was thrown when the migrate_schemas command was called in models.py around line 114
